cpp/.vcpkg-overlay/intel-mpi/portfile.cmake

```diff
@@ -0,0 +1,78 @@
+set(INTELMPI_VERSION "2021.12")
```

Comment:
Could you add a comment to summarise what this file does? What are the prerequisites?

Reply:
Added README.md.